x/finality/keeper/tallying.go

```diff
@@ -19,7 +19,7 @@ import (
 // but without block that has finality providers set AND does not receive QC
 func (k Keeper) TallyBlocks(ctx context.Context) {
 	sdkCtx := sdk.UnwrapSDKContext(ctx)
-	activatedHeight, err := k.BTCStakingKeeper.GetBTCStakingActivatedHeight(ctx)
+	activatedHeight, err := k.GetBTCStakingActivatedHeight(ctx)
```

Comment:
Maybe we should call `k.IsFinalityActive()` here?

Reply:
Seems already checked at the caller's side

Reply:
Yep this function (together with `IndexBlock` and `HandleLiveness`) will be invoked only when `IsFinalityActive` returns true